application/models/Question.php

```diff
@@ -89,6 +89,9 @@ class Question extends LSActiveRecord
     /** Set defaut relevance **/
     public $relevance ='';
 
+    /** @var QuestionTheme cached question theme*/
+    private $relatedQuestionTheme;
```

Comment:
Isn't the relation already cached by Yii...? o0

Reply:
https://www.yiiframework.com/doc/guide/1.1/en/caching.data

Reply:
Maybe we should not say cache. Maybe a temporary storage for avoiding round trips.

> Isn't the relation already cached by Yii...? o0

We are not using the relation as when creating questions, that relation is not loaded.
So we are using a method that mimics it.

Reply:
Technically correct term: https://en.wikipedia.org/wiki/Memoization